src/content/fr/2019/seo.md

```diff
@@ -0,0 +1,421 @@
+  </a>
+  <div id="fig10-description" class="visually-hidden">Distribution des performances des expériences utilisateur des ordinateurs de bureau, des téléphones et des tablettes. Ordinateur de bureau: 2% rapide, 52% modéré, 46% lent. Téléphone: 1% rapide, 41% modéré, 58% lent. Tablette: 0% rapide, 35% modérée, 65% lente.</div>
+  <figcaption id="fig10-caption">Figure 10. Distribution of the performance of user experiences by device type.</figcaption>
+</figure>
+
+Segmentée par type d'appareil, cette image est encore plus sombre pour la tablette (65%) et le mobile (58%).
+Bien que les chiffres soient inquiétants pour la vitesse du web, la bonne nouvelle est que les experts et les outils SEO se concentrent de plus en plus sur les défis techniques de l'accélération des sites. Vous pouvez en savoir plus sur l'état des performances web dans le chapitre [Performances](./performances).
+
+### Données structurées
+
+Les données structurées permettent aux propriétaires de sites web d'ajouter des données sémantiques supplémentaires à leurs pages web, en ajoutant des extraits de code [JSON-LD](https://en.wikipedia.org/wiki/JSON-LD) ou des [microdonnées](https://developer.mozilla.org/fr/docs/web/HTML/Microdonn%C3%A9es), par exemple. Les moteurs de recherche analysent ces données pour mieux comprendre ces pages et utilisent parfois le balisage pour afficher des informations pertinentes supplémentaires dans les résultats de la recherche. Les types de données structurées les plus courants sont:
+- [Review](https://developers.google.com/search/docs/data-types/review-snippet)
+- [Product](https://developers.google.com/search/docs/data-types/product)
+- [Local Bussiness](https://developers.google.com/search/docs/data-types/local-business)
+- [Movie](https://developers.google.com/search/docs/data-types/movie)
+- et [D'autres](https://developers.google.com/search/docs/guides/search-gallery)
+La [visibilité supplémentaire](https://developers.google.com/search/docs/guides/enhance-site) que les données structurées peuvent fournir aux sites web est intéressante pour les propriétaires de sites, car elle peut aider à créer plus d'opportunités de trafic . Par exemple, le [schéma de FAQ] relativement nouveau (https://developers.google.com/search/docs/data-types/faqpage) doublera la taille de votre extrait sur les pages de résultats de recherche.
+Au cours de nos recherches, nous avons constaté que seuls 14,67% des sites sont éligibles pour des résultats riches sur mobile. Fait intéressant, l'admissibilité au site de bureau est légèrement inférieure à 12,46%. Cela suggère que les propriétaires de sites peuvent faire beaucoup plus pour optimiser la façon dont leurs pages d'accueil apparaissent dans la recherche.
+Parmi les sites avec un balisage de données structuré, les cinq types les plus implémentés sont:
+1. «webSite» (16,02%)
+2. «SearchAction» (14,35%)
+3. «Organisation» (12,89%)
+4. `webPage` (11,58%)
+5. «ImageObject» (5,35%)
```

Comment:
Is this formatting intentional? Presume not?

```suggestion
1. `WebSite` (16,02&nbsp;%)
2. `SearchAction` (14,35&nbsp;%)
3. `Organisation` (12,89&nbsp;%)
4. `WebPage` (11,58&nbsp;%)
5. `ImageObject` (5,35&nbsp;%)
```

Reply:
They are not code...Name of Schema types but ok.